daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state copy.md

```diff
@@ -0,0 +1,95 @@
+---
+type: docs
+title: "How-To: Change a state sharing strategy"
+linkTitle: "How-To: Change a state sharing strategy"
+weight: 400
```

Comment:
Why does this file have "copy" in the name?

Reply:
Good catch, this should be changed to `howto-change-state-sharing.md`

Reply:
changed